Jenkinsfile

```diff
@@ -0,0 +1,576 @@
+        registry_port = "5000"
+
+        start_time = sh(script: 'date +"%Y-%m-%d %H:%M:%S"', returnStdout: true).trim()
+        wle_download_url="https://github.com/oracle/weblogic-logging-exporter/releases/latest"
+        wko_build_email='robert.patrick@oracle.com'
+    }
+
+    parameters {
+        string(name: 'IT_TEST',
+               description: 'Comma separated ItClasses to be run e.g., ItParameterizedDomain, ItMiiUpdateDomainConfig, ItMiiDynamicUpdate*, ItMiiMultiMode',
+               defaultValue: '**/It*'
+        )
+        choice(name: 'MAVEN_PROFILE_NAME',
+               description: 'Profile to use in mvn command to run the tests.  Possible values are integration-tests, toolkits-srg, kind-sequential, wls-image-cert, fmw-image-cert, and fmw-srg.  Refer to weblogic-kubernetes-operator/integration-tests/pom.xml on the branch.',
+               choices: [
+                   'integration-tests',
+                   'toolkits-srg',
+                   'kind-sequential',
+                   'wls-image-cert',
+                   'fmw-image-cert',
+                   'fmw-srg'
+               ]
+        )
+        choice(name: 'KIND_VERSION',
+               description: 'Kind version.',
+               choices: [
+                   '0.11.1',
+                   '0.11',
+                   '0.10',
+                   '0.9',
+                   '0.8'
+               ]
+        )
+        choice(name: 'KUBE_VERSION',
+               description: 'Kubernetes version. Supported values depend on the Kind version.  Kind 0.9.0: 1.19, 1.19.1, 1.18, 1.18.8, 1.17, 1.17.11, 1.16, 1.16.15, 1.15, and 1.15.12.  Kind 0.10.0: 1.20, 1.20.2, 1.19, 1.19.7, 1.18, 1.18.15, 1.17, 1.17.17, 1.16, and 1.16.15.  Kind 0.11.0 and 0.11.1: 1.23, 1.23.0, 1.22, 1.22.0, 1.21, 1.21.1, 1.20, 1.20.7, 1.19, 1.19.11, 1.18, 1.18.19, 1.17, 1.17.17, 1.16, and 1.16.15.',
+               choices: [
+                    // The first item in the list is the default value...
+                    '1.21.1',
+                    '1.23.3',
+                    '1.23',
+                    '1.22.5',
+                    '1.22',
+                    '1.21',
+                    '1.20.7',
+                    '1.20',
+                    '1.19.11',
+                    '1.19',
+                    '1.18.19',
+                    '1.18',
+                    '1.17.17',
+                    '1.17',
+                    '1.16.15',
+                    '1.16',
+                    '1.15.12',
+                    '1.15'
+               ]
+        )
+        string(name: 'KUBECTL_VERSION',
+               description: 'kubectl version',
+               defaultValue: '1.21.5'
+        )
+        string(name: 'HELM_VERSION',
+               description: 'Helm version',
+               defaultValue: '3.7.2'
+        )
+        string(name: 'ISTIO_VERSION',
+               description: 'Other Possible Values 1.7.3, 1.8.1, 1.7.6',
+               defaultValue: '1.10.4'
```

Comment:
I thought that we were defaulting to 1.11 now

Reply:
I just copied the default from the current job.  We can change this, if desired.

Reply:
Okay, I don't think this should hold up your PR. It will be an ongoing activity to update defaults.